src/test/java/uk/gov/pay/ledger/transaction/resource/TransactionResourceSearchIT.java

```diff
@@ -408,4 +408,42 @@ public void shouldSearchCorrectlyOnTransactionAndParentTransaction_WhenWithParen
                 .body("results[1].transaction_id", is(payment.getExternalId()))
                 .body("results[1].parent_transaction", is(nullValue()));
     }
+
+    @Test
+    public void shouldSearchUsingMultipleGatewayAccountIdsExclusively() {
```

Comment:
I don’t think this needs `Exclusively` in the name now the other test has been removed.